package.json

```diff
@@ -4,11 +4,9 @@
   "description": "minted-frontend-boilerplate",
   "main": "index.js",
   "scripts": {
-    "build-client": "webpack --config webpack.client.js",
     "build-server": "webpack --config webpack.server.js",
     "build-app-renderer-server": "webpack --config webpack.appRendererServer.js",
-    "build": "yarn run build-client && yarn run build-server && yarn build-app-renderer-server",
-    "dev": "node build/server.js"
```

Comment:
We could name these `build:client` and `build:server`, so it similar to other projects.

But I think there's not a situation where we want to individually build them out. Correct me if I'm wrong, but I think we want 2 commands:
1. Local development.
2. Building out the 2 bundles App-Renderer needs (client and app-renderer-server) which get published to S3.

Am I missing anything? Ah yes, unit tests :p. Adding Cypress and Jest commands would also be useful.

If you look at what I did for OneForm (https://github.com/Sawtaytoes/OneForm/blob/master/packages/react/package.json), I set it up so there are separate commands, but I use the `npm-run-all` library to run them all together for the base `build` and `dev` commands.